Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -473,7 +473,8 @@ - (void)pdfLayerController:(PDFLayerController *)pdfLayerController didChangeDis
 
 - (void)pdfLayerController:(PDFLayerController *)pdfLayerController didChangeSelection:(PDFSelection *)selection
 {
-    _pdfPlugin->notifySelectionChanged(selection);
+    UNUSED_PARAM(selection);
```

Comment:
You may not need this. Harmless but ugly.

Reply:
Fair, it's not necessary. I'll drop it.

Reply:
Removed in [d46739f](https://github.com/WebKit/WebKit/commit/d46739f2c991aac30310fc1769aac10eda429874)